08-communicating-results.Rmd

```diff
@@ -89,12 +91,12 @@ We could also make the data more readable in this table by formatting the column
 ```{r}
 #| label: results-table-fix2
 bill_region_gt2 <- bill_region_gt %>%
-  fmt_currency(columns = elec_bill,
-               decimals = 0,
-               currency = "USD") %>%
-  fmt_currency(columns = elec_bill_se,
-               decimals = 4,
-               currency = "USD") %>%
+  fmt_currency(columns = elec_bill, 
+               decimals = 0, 
+               currency = "USD") %>% 
+  fmt_currency(columns = elec_bill_se, 
+               decimals = 4, 
+               currency = "USD") %>% 
```

Comment:
This now wends up in the billions of dollars. Before you were just counting cases using is.na. Might want to consider using scale_by=1/1000000 to make it in the millions and update header appropriately.